packages/core/components/Modal/MoveFileManifest/index.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react";
+import { useSelector } from "react-redux";
+
+import { ModalProps } from "..";
+import BaseModal from "../BaseModal";
+import { PrimaryButton } from "../../Buttons";
+import { interaction, selection } from "../../../state";
+import FileDetail from "../../../entity/FileDetail";
+import styles from "./MoveFileManifest.module.css";
```

Comment:
The ordering here is off:
```
import { ModalProps } from "..";
import BaseModal from "../BaseModal";
import { PrimaryButton } from "../../Buttons";
import FileDetail from "../../../entity/FileDetail";
import FileSelection from "../../../entity/FileSelection";
import { interaction, selection } from "../../../state";

import styles from "./MoveFileManifest.module.css";
```

Reply:
resolved in [1201a84](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/1201a846f67e846b16e1c4adc2b738813713e2f7)